src/main/scala/scalaz/ziokeeper/crdt/ORSet.scala

```diff
@@ -0,0 +1,31 @@
+package scalaz.ziokeeper.crdt
```

Comment:
The new root package will probably be:

```scala
package dev.zio.keeper
```

Or maybe just:

```scala
package zio.keeper
```

Reply:
Shall we migrate the project to new organization?

Reply:
Yes, no rush on that, but to avoid cluttering `scalaz` we should get all the active ZIO projects to the new org (and delete the remainder). 👍

Reply:
In that case, let's polish this PR, and I'll take care of the migration and packages cleanup.